app/models/host/managed.rb

```diff
@@ -631,6 +631,11 @@ def set_ip_address
     if SETTINGS[:unattended] && (new_record? || managed?)
       self.ip  ||= subnet.unused_ip.suggest_ip if subnet.present?
       self.ip6 ||= subnet6.unused_ip.suggest_ip if subnet6.present?
+      self.interfaces.each do |nic|
+        next if nic.primary
```

Comment:
self.ip is delegated to the primary interface. I think, a better approach is to just loop over all interfaces and leave out the "next" line.
```ruby
self.interfaces.each do |nic|
  nic.ip  ||= nic.subnet.unused_ip.suggest_ip if nic.subnet.present?
  nic.ip6 ||= nic.subnet6.unused_ip.suggest_ip if nic.subnet6.present?
end
```